_pydevd_sys_monitoring/_pydevd_sys_monitoring.py

```diff
@@ -1841,7 +1843,10 @@ def update_monitor_events(suspend_requested: Optional[bool] = None) -> None:
         monitor.register_callback(DEBUGGER_ID, monitor.events.PY_START, _start_method_event)
         # monitor.register_callback(DEBUGGER_ID, monitor.events.PY_RESUME, _resume_method_event)
         monitor.register_callback(DEBUGGER_ID, monitor.events.LINE, _line_event)
-        monitor.register_callback(DEBUGGER_ID, monitor.events.JUMP, _jump_event)
+        if not IS_PY313_OR_GREATER:
```

Comment:
We get different line_event output from 3.13 than we did in 3.12. Jump events are now completely redundant. The tests around looping were failing without this change.